implementations/rust/xeddsa/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "xeddsa"
+version = "0.1.0"
+authors = ["Michael Lodder <mike@ockam.io>"]
```

Comment:
I've seen us standardize on "Ockam Developers" as the only author entry -- not opposed to start tracking who has worked on these crates and keep the authors entry updated.

Reply:
good catch. I'll make the change.